packages/compass-saved-aggregations-queries/src/hooks/use-grid-filters.tsx

```diff
@@ -219,7 +218,7 @@ export function filterByText(items: Item[], text: string): FilterItem[] {
         }
       }
 
-      return item[key as keyof Item] as any;
+      return item[key];
```

Comment:
Noticed that we are using `any` here and had to change the code a bit to make sure we don't need to